CI/macos/01_install_dependencies.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+
+##############################################################################
+# macOS dependency management function
+##############################################################################
+#
+# This script file can be included in build scripts for macOS or run directly
+#
+##############################################################################
+
+# Halt on errors
+set -eE
+
+install_obs-deps() {
+    status "Setup for pre-built macOS OBS dependencies v${1}"
```

Comment:
"Setup" is a noun.  "Set up" is a verb.  I'm not sure which is intended here, but "Setup" is used throughout this and other scripts.  If the verb form is intended, "Set up X" rather than "Set up for X" is fine.

Reply:
Fixed.